deploy/charts/cert-manager/templates/cainjector-poddisruptionbudget.yaml

```diff
@@ -26,4 +26,9 @@ spec:
   {{- if hasKey .Values.cainjector.podDisruptionBudget "maxUnavailable" }}
   maxUnavailable: {{ .Values.cainjector.podDisruptionBudget.maxUnavailable }}
   {{- end }}
+  {{- if (semverCompare ">= 1.27-0" .Capabilities.KubeVersion.Version) }}
```

Comment:
I think this conditional can be removed. Even on Kubernetes 1.27 this requires enabling a feature gate. The feature gate is only enabled by default from K8s 1.31, ref. https://kubernetes.io/docs/tasks/run-application/configure-pdb/#unhealthy-pod-eviction-policy

Same in other places.